src/3box.js

```diff
@@ -286,8 +286,7 @@ class Box extends BoxApi {
    */
   get DID () {
     if (!this._3id) throw new Error('DID: auth required')
-    // TODO - update once verification service supports 3ID
-    return this._3id.muportDID
+    return this._3id.did
```

Comment:
Pretty sure this needs to be `this._3id.DID`.

Reply:
```suggestion
    return this._3id.DID
```
Had a look. It definitely needs to be capitalized.